pkgs/top-level/all-packages.nix

```diff
@@ -15091,6 +15091,10 @@ in
 
   haka = callPackage ../tools/security/haka { };
 
+  hasura-graphql-engine = (callPackage ../servers/hasura { }).graphql-engine;
+
+  hasura-cli = (callPackage ../servers/hasura { }).cli;
```

Comment:
I'd suggest
```
inherit (callPackage ../servers/hasura { })
  hasura-cli
  hasura-graphql-engine
;
```
We already have precendents of this style in `all-packages.nix`. But sure, this is optional.

Reply:
Okay, i did suggested change